### PR TITLE
Skip build optimizations in dev env. Fixes #1406

### LINKED ISSUE #1406: disable postinstall minification in dev

### PR DESCRIPTION
**Pull request by brollb**

### CODE REVIEW

File: bin/build.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+/* eslint-disable no-console */
 'use strict';
 
+const isDevEnv = process.env.ENV === 'dev';
```
Comment (hamidzr) [inline bin/build.js]: should we consider lack of `production` environment dev instead?

Comment (hamidzr) [inline bin/build.js]: `const isDevEnv = process.env.ENV !== 'production';`

**Comment by brollb** [comment]
I thought it would be better to put a more restrictive check on this step
to make sure it errs on the side of generating the optimized scripts.
However, for consistency, it might be best to change it as you suggested

**Comment by hamidzr** [comment]
makes sense. we also have this `local-dev`
```
src/server/server.js
17:        vantage: process.env.ENV !== 'production'
63:        if (validOrigins.test(origin) || process.env.ENV === 'local-dev') {
128:        if(process.env.ENV !== 'production'){

bin/build.js
4:const isDevEnv = process.env.ENV === 'dev';

bin/netsblox-start.js
26:    vantage: process.env.ENV !== 'production'

```

**Comment by brollb** [comment]
yeah, I might change this to check that the env is not `production` rather than if it is dev

**Pull request merged**